packages/assets-controllers/src/NftController.ts

```diff
@@ -520,13 +520,13 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
 
     return {
       ...openSeaMetadata,
-      name: blockchainMetadata.name ?? openSeaMetadata?.name ?? null,
+      name: blockchainMetadata?.name ?? openSeaMetadata?.name ?? null,
```

Comment:
@bergeron How would you describe the impact of these changes? It looks like they'd have a functional impact.

This suggests that there may have been cases where `blockchainMetadata` was nullish, which previously would have thrown an error on this line. But now it's no longer thrown.

Similarly, below in `getNftContractInformation`, the changes suggest that `address` and `contract.name` would previously be missing sometimes, but now they're always included.

Is that the case? Or are the affected scenarios really non-existent, with these changes being made just for type reasons.

Reply:
It looks like this does fix something. In the case where the request fails, this would crash on this line previously, but now it doesn't.

Too bad this isn't represented in tests either, there are no tests for this method. And even indirect uses of it are stubbed out in the tests :facepalm: